io.github.piotrek_k._2dTaskBoard.yml

```diff
@@ -0,0 +1,62 @@
+app-id: io.github.piotrek_k._2dTaskBoard
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node20
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
```

Comment:
I needed to add this permission, because after I installed the build provided today by flathubbot, an app has been showing blank screen. After I added this permission it started to work again.

Reply:
it's fine